src/test/java/edu/cornell/kfs/concur/eventnotification/rest/jsonObjects/ConcurDTOJsonTestConstants.java

```diff
@@ -0,0 +1,17 @@
+package edu.cornell.kfs.concur.eventnotification.rest.jsonObjects;
+
+public class ConcurDTOJsonTestConstants {
+    public static final String OAUTH2_TOKEN_RESPONSE_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/jsonObjects/fixture/ConcurOauth2TokenResponse.json";
+    public static final String OAUTH2_TOKEN_RESPONSE_WITH_EXTRA_VALUES_FILE_NAME = "src/test/resources/edu/cornell/kfs/concur/rest/jsonObjects/fixture/ConcurOauth2TokenResponseWithExtraValues.json";
+    
+    public static final class Oauth2TokenResponseValue {
```

Comment:
Some of the constants in this nested class are quite large. Is there a better way to externalize their values to a file? If that ends up being too much trouble, then I'm okay with leaving them in this class for now.